R/outputs.R

```diff
@@ -977,8 +980,17 @@ save_output <- function(filename, dir,
   }
 
   if (export_datapack) {
+
+    if (!is.null(vmmc_path)) {
+      assert_scalar_character(vmmc_path)
+      ## Skip the first row, the file has two rows of headers
+      vmmc_datapack <- openxlsx::read.xlsx(vmmc_path, sheet = "Datapack inputs",
+                                           startRow = 2)
```

Comment:
How do we want this to fail if sheet is not found?

Currently it will return an error from `openxlsx`: `Cannot find sheet named "Datapack inputs"`

Reply:
Good spot, just added a better check to hintr for this case. So users will see an error on file upload. See https://github.com/mrc-ide/hintr/pull/485/commits/5a56a2d31b10dbcc0624e4f3bc0d5544d077cf6b

So with that, we could make this check better in the same vein as the hintr commit. So this message will be unlikely to show for users of the web app now, but will be shown if running it via R.

Reply:
Will leave this for addressing in #424